apps/desktop/src/lib/branch/SeriesHeader.svelte

```diff
@@ -165,6 +166,28 @@
 		}
 	});
 
+	/**
+	 * If the repository does not have "delete after merged" enabled, we need to manually update the base
+	 * of the bottom most branch to point to the projects baseBranch of choice after its parent has been integrated,
+	 * and  it now is the bottom-most series.
+	 */
+	$effect(() => {
```

Comment:
Not super happy with putting this here, open to any other suggestions

Reply:
I'm always a bit worried about effects that cover many different reactive elements. And.. currently I'm wondering, what would happen if the update call was rejected with a 401? It feels like we could end up with lots of retries, and worst some loop.

Is there anything we could do to mitigate that?

Reply:
Yeah good call, I added a `catch` to the then chain and a boolean to stop looping